pkg/registry/common/connect/server_ns_test.go

```diff
@@ -0,0 +1,144 @@
+package connect_test
+
+import (
+	"context"
+	"github.com/networkservicemesh/api/pkg/api/registry"
+	"github.com/networkservicemesh/sdk/pkg/registry/common/connect"
+	"github.com/networkservicemesh/sdk/pkg/registry/common/dnsresolve"
+	"github.com/networkservicemesh/sdk/pkg/registry/common/swap"
+	"github.com/networkservicemesh/sdk/pkg/registry/core/next"
+	"github.com/networkservicemesh/sdk/pkg/registry/memory"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
+	"testing"
+)
+
+func localNSRegistryServer(proxyDomain string, resolver dnsresolve.Resolver) registry.NetworkServiceRegistryServer {
+	return next.NewNetworkServiceRegistryServer(
+		memory.NewNetworkServiceRegistryServer(),
+		dnsresolve.NewNetworkServiceRegistryServer(
```

Comment:
@edwarnicke I've added dnsresolve chain element for the local registry to find proxy registry in the cluster. 
Question: Should local registry use DNS?  If not then how local registry should find proxy registry?